### PR TITLE
make some changes for launch

### PR DESCRIPTION
**Pull request by ladyleet**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- **What is the current behavior?** (You can also link to an open issue here)
- **What is the new behavior (if this is a feature change)?**
- **Other information**:

### CODE REVIEW

**Comment by ladyleet** [comment]
👍 @gsans ? / @benlesh

**Comment by ladyleet** [comment]
BEN WHY ARE YOU NOT TAGGABLE

**Comment by ladyleet** [comment]
@JAYPHELPS please lint my code.

**Comment by jayphelps** [comment]
@ladyleet for reasons unknown, Ben is @blesh on Github, not @benlesh

**Comment by ladyleet** [comment]
@blesh 👍 ?

**Comment by ladyleet** [comment]
@gsans 👍 💯 ?

**Comment by ladyleet** [comment]
@jayphelps ohai

**Comment by ladyleet** [comment]
@jayphelps i'm practicing bikeshedding like you taught me.

File: src/app/home/components/welcome.ts [hunk 1]
```diff
@@ -5,13 +5,15 @@ import { Component } from '@angular/core';
...
       <div class="welcome-sub">
-        Sub Headline Lorem Ipsum Dolor Sit<br/>
+        <p>As <a href="https://twitter.com/jeffbcross/status/776891439657525250">Jeff Cross' beard</a> has stated,</p>
+        <p>we're officially jumping the shark</p>
+        <p>May 29, 2017!</p>
         <a href="https://www.ng-conf.org/#/conduct" title="This code of conduct was adopted from ng-conf's code of conduct" target="_blank">Code of Conduct</a>
       </div>
       <div class="centerBlock">
-        <button class="main-btn grow" onclick="document.location='/#/home#tickets';return false;">Purchase</button> 
+        <button class="main-btn grow" onclick="document.location='/#/home#tickets';return false;">Purchase</button>
```
Comment (jayphelps) [inline src/app/home/components/welcome.ts:16]: I'm no ng2 pro--but I'm guessing this isn't the idiomatic way to route to a different page

Comment (ladyleet) [reply src/app/home/components/welcome.ts:16]: oh the sheds of the bikes @jayphelps. i blame @gsans

File: src/app/home/components/speakers.ts [hunk 2]
```diff
@@ -20,7 +23,7 @@ import { Component } from '@angular/core';
           Miško is the creator of the Angular framework. He has passion for making complex things simple. He currently works at Google, but has previously worked at Adobe, Sun Microsystems, Intel, and Xerox, where he became an expert in building web applications in web related technologies such as Java, JavaScript, Flex and ActionScript.
         </profile>
       </div>
-    </div>
+    </div> -->
```
Comment (jayphelps) [inline src/app/home/components/speakers.ts:26]: This section is commented out..is that desired? Why not remove it?

**Comment by jayphelps** [comment]
Overall solid.

**Comment by ladyleet** [comment]
@gsans you gerard you want to merge my pr?

**Comment by ladyleet** [comment]
merged my own.

**Comment by ladyleet** [comment]
REBASE FOR ALL THE REBASE GODS! @BLESH @jayphelps @gsans

**Comment by benlesh** [comment]
Sneaky Clooney

**Comment by gsans** [comment]
I'm writing this comment naked as a tribute to "probably" naked profile pics =)

**Comment by ladyleet** [comment]
@gsans NO YOU DIDNT.

**Comment by gsans** [comment]
Well... I'll leave that to your imagination =)

**Pull request merged**